source/Halibut/Diagnostics/HalibutTimeoutsAndLimits.cs

```diff
@@ -153,7 +153,16 @@ public TimeSpan SafeTcpClientPooledConnectionTimeout
         /// The duration a TCP connection will wait for a keepalive response before sending another keepalive probe.
         /// </summary>
         public TimeSpan TcpKeepAliveInterval { get; set; } = TimeSpan.FromSeconds(5);
-        
+
+        /// <summary>
+        /// The maximum number of active TCP connections per polling subscription. <c>null</c> indicates there is no limit.
+        /// </summary>
+        /// <remarks>
+        /// This setting is used to prevent denial-of-service/connection exhaustion due to too many incoming connections from a single polling subscription.
+        /// The number of authorized, active connections are aggregated per polling subscription, and new connections that exceed the limit are rejected.
```

Comment:
nice doco, also now that you changed to poll address over ssl thumbprint its way easier to understand.